test/clangsa/MissingRoots.c

```diff
@@ -67,20 +67,6 @@ void root_value_data2() {
 
 
 void root_value_data3() {
-    jl_svec_t *val = jl_svec1(NULL); // expected-note{{Started tracking value here}}
-    jl_value_t **data = jl_svec_data(val);
-    JL_GC_PUSH1(&val); // expected-note{{GC frame changed here}}
-                       // expected-note@-1{{Value was rooted here}}
-    jl_gc_safepoint();
-    look_at_value(**&data);
-    JL_GC_POP(); // expected-note{{GC frame changed here}}
-                 // expected-note@-1{{Root was released here}}
-    jl_gc_safepoint(); // expected-note{{Value may have been GCed here}}
-    **&data; // expected-warning{{Creating derivative of value that may have been GCed}}
-             // expected-note@-1{{Creating derivative of value that may have been GCed}}
```

Comment:
Why remove this test?

Reply:
It doesn't pass anymore (I think it's too much state to explore now?), but also doesn't seem too realistic.

Reply:
Ok, that seems odd, because the state should be fairly limited here, but I'm ok dropping this test if you need to make progress.

Reply:
Yeah—I had added this test just for the purpose of testing this PR, so I'm okay with removing it.